src/Form/selectors.js

```diff
@@ -132,21 +134,21 @@ const CC_MATCHERS_LIST = [
         type: 'expirationMonth',
         selector: CC_MONTH_SELECTOR,
         matcherFn: (string) =>
-            /(card|cc)?.?(exp(iry|iration)?)?.?(month|mm(?![.\s/-]yy))/i.test(string) &&
+            /(card|\bcc\b)?.?(exp(iry|iration)?)?.?(month|\bmm\b(?![.\s/-]yy))/i.test(string) &&
```

Comment:
Adding `\b` (word boundaries) to minimise false matches (for example "a**cc**ept" and "co**mm**it").